packages/seacas/applications/explore/CMakeLists.txt

```diff
@@ -20,13 +20,17 @@ TRIBITS_ADD_EXECUTABLE(
   LINKER_LANGUAGE Fortran
   SOURCES ${SOURCES}
   COMM serial mpi
+  ADDED_EXE_TARGET_NAME_OUT explore_added
```

Comment:
Are these changes to seacas going to be erased on the next snapshot over?

Reply:
@rppawlo 

> Are these changes to seacas going to be erased on the next snapshot over?

As soon as @gsjaardema merges https://github.com/gsjaardema/seacas/pull/154 we will be set.